gbpservice/nfp/configurator/lib/demuxer.py

```diff
@@ -105,45 +103,56 @@ def get_service_agent_info(self, operation, service_type, request_data):
         """
 
         sa_info_list = []
+        vendor_map = {const.FIREWALL: const.VYOS,
+                      const.LOADBALANCER: const.HAPROXY}
+
+        service_vendor = request_data['info']['service_vendor']
+        if str(service_vendor) == 'None':
```

Comment:
Are we not mandating service_vendor? What are the cases where we get None for service vendor?